tests/.tests-config.tpl.js

```diff
@@ -10,14 +10,16 @@ export default {
   },
   credentials: { // commented companies will be skipped automatically, uncomment those you wish to test
     // hapoalim: { userCode: '', password: '' },
-    // leumi: { username: '', password: '' },
+    // hapoalimBeOnline: { userCode: '', password: '' },
```

Comment:
@dratler this line has been removed in https://github.com/eshaham/israeli-bank-scrapers/commit/f0f0a8c682260619464b4c16c3d9ab6d07f24206#diff-66471da957486cfa9ae4dda19b4f7e96
could you please remove it?

Reply:
done

Reply:
@dratler weird, I'm still seeing it in the PR